### PR TITLE
Break out tables into unnumbered sections. Use verbatim in examples

### PR DESCRIPTION
**Pull request by rocky**
The simple instructions that were just added need to be converted to unnumbered sections, but I though I should get this in for review before I go too far down that road.

### CODE REVIEW

**Comment by pipcet** [comment]
That looks very good to me, though I notice we're inconsistent between `byte-var-ref` and `byte-varref`.

Also, I think some of the examples assume dynamic binding, and we should note that.

File: elisp-bytecode.texi [hunk 1]
```diff
@@ -59,22 +59,35 @@ A stack reference
...
 @unnumberedsubsec @code{byte-varset} (16--23)
 @kindex byte-varset
 
-Sets a variable
+Sets a variable given in the operand to the value that is on the top
+of the stack.
+
+@subsubsection Example
+
+@code{(defun en(n5) (setq n 5)} generates:
```
Comment (pipcet) [inline elisp-bytecode.texi]: I'm just noticing because I'm putting together a function to re-generate the examples, but we're missing a `)` here.

**Comment by rocky** [comment]
Ok will address in next PR.

File: elisp-bytecode.texi (hunk 1 above)
Comment (pipcet) [reply elisp-bytecode.texi]: Also, the `n5` argument is unused, which is a little confusing.

Comment (rocky) [reply elisp-bytecode.texi]: Ok. Both of these are in the next pull request.

**Pull request merged**